mypy/semanal.py

```diff
@@ -6408,14 +6408,44 @@ def lookup_fully_qualified_or_none(self, fullname: str) -> SymbolTableNode | Non
         # TODO: support nested classes (but consider performance impact,
         #       we might keep the module level only lookup for thing like 'builtins.int').
         assert "." in fullname
-        module, name = fullname.rsplit(".", maxsplit=1)
-        if module not in self.modules:
+
+        splitted_modules = fullname.rsplit(".")
```

Comment:
Minor: Here you can use just `fullname.split(".")`, since it won't make a difference in semantics and `split` is the more usual variant.